modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/accessor/UserRoleAccessor.java

```diff
@@ -28,6 +28,9 @@
 @Accessor
 public interface UserRoleAccessor {
 
+    @Query("select * from user_roles")
+    Result<UserRole> getAllUserRoles();
+
     @Query("select * from user_roles where user_id = ? and app_name = ?")
     Result<UserRole> getUserRolesBy(String userId, String appName);
```

Comment:
@abhishekjain88 I see we are using userID more often in the code. It will be more consistent to change userId to userID. just a suggestion.